ExpenseMaster.Model/Models/Budget.cs

```diff
@@ -0,0 +1,15 @@
+﻿
+namespace ExpenseMaster.Model.Models
+{
+    public class Budget
+    {
+        public int Id { get; set; }
+        int UserId {get; set;}
+        int CategoryId { get; set; }
+        public decimal Limit { get; set; }
+        public decimal WarningThreshold { get; set; }
+
+        public User? User { get; set; }
```

Comment:
And it is better to write entity and id of this entity together to make code more readable